drivers/rackspace/rackspace.go

```diff
@@ -62,6 +63,16 @@ func GetCreateFlags() []cli.Flag {
 			EnvVar: "OS_FLAVOR_ID",
 		},
 		cli.StringFlag{
+			Name:  "rackspace-net-id",
+			Usage: "Rackspace comma separated network ids the machine will be connected on",
+			Value: "",
+		},
+		cli.StringFlag{
```

Comment:
Funny, I actually had that and changed it to match the implementation of `openstack-sec-groups`. Should we make that change to also include it?  Is there anyone who can test that switch if we do?